build-time-analytics/src/test/resources/remote-config-disabled.json

```diff
@@ -0,0 +1,6 @@
+{
+  "active": false,
+  "deny_anonymous_ids": [],
+  "deny_quarkus_versions": [],
+  "refresh_interval": 43200.000000000
```

Comment:
No, jutst a default value generated by the quarkus implementation. It's not being used here.

Reply:
Actually, I was wrong. It is being used, not by the tests, but by Quarkus. It says how frequently should this remote config be updated from remote source. The value here is Quarkus' default (in seconds - 12 hours).

Since this is a local resource used for testing, I'll change the value from the default, which might look suspicious, to something clearly artificial.
We also want to make sure that this config does not get refreshed during the test execution, so the value should be big enough.

Let's make it `999999`, which is bigger than the already big default and looks more intentional.

Furthermore, I forgot to test the refresh feature itself, so I'll add 2 new tests of that (in DEV mode and JVM mode).